go.mod

```diff
@@ -6,3 +6,7 @@ require (
 	github.com/dome9/dome9-sdk-go v1.16.0
 	github.com/hashicorp/terraform-plugin-sdk v1.1.0
 )
+replace (
```

Comment:
Don't forget to clean this up and update the SDK version instead (once published)